src/cli/simple/command.rs

```diff
@@ -486,6 +491,117 @@ pub async fn paused_precompiles(client: Client) -> anyhow::Result<()> {
     get_value::<u32>(client, "paused_precompiles", None).await
 }
 
+///////////////////////////////////////////////////////////////
+//                     SILO SPECIFIC METHODS                 //
+///////////////////////////////////////////////////////////////
+
+// Return fixed gas cost
+pub async fn get_fixed_gas_cost(client: Client) -> anyhow::Result<()> {
+    get_value::<U256>(client, "get_fixed_gas_cost", None).await
+}
+
+pub async fn set_fixed_gas_cost(client: Client, cost: u64) -> anyhow::Result<()> {
+    let args = Wei::new_u64(cost).try_to_vec()?;
+
+    ContractCall {
+        method: "set_fixed_gas_cost",
+        success_message: "Fixed gas cost was set successfully",
+        error_message: "Error while setting gas cost",
+    }
+    .proceed(client, args)
+    .await
+}
+
+pub async fn get_whitelist_status(client: Client, whitelist_kind: String) -> anyhow::Result<()> {
+    let args = WhitelistKindArgs {
+        kind: get_kind(whitelist_kind)?,
+    }
+    .try_to_vec()?;
+
+    get_value::<U256>(client, "get_whitelist_status", args).await
+}
+
+pub async fn set_whitelist_status(
+    client: Client,
+    whitelist_kind: String,
+    status: bool,
+) -> anyhow::Result<()> {
+    let args = WhitelistStatusArgs {
+        kind: get_kind(whitelist_kind)?,
+        active: status,
+    }
+    .try_to_vec()?;
+
+    ContractCall {
+        method: "set_whitelist_status",
+        success_message: "Set whitelist status successfully",
+        error_message: "Error while setting whitelist status",
+    }
+    .proceed(client, args)
+    .await
+}
+
+pub async fn add_entry_to_whitelist(
+    client: Client,
+    whitelistArgs: String,
+    kind: String,
+    address: String,
+) -> anyhow::Result<()> {
+    let args = get_whitelist_args(client, whitelistArgs, kind, address)?;
+
+    ContractCall {
+        method: "add_entity_to_whitelist",
+        success_message: "Added entity to whitelist successfully",
+        error_message: "Error while adding entity to whitelist",
+    }
+    .proceed(client, &args)
+    .await
+}
+
+pub async fn add_entry_to_whitelist_batch(
```

Comment:
I guess we should provide a batch of entries by passing a path to a JSON file. And the file should have a format like this:
```json
[
    {
        "kind": "Admin",
        "account_id": "account.near"
    },
    {
        "kind": "Address",
        "address": "0x592186c059e3d9564cac6b1ada6f2dc7ff1d78e9"
    }
]
```
What you need is:
1. Deserialize the file's content to `Vec<WhitelistArgs>`.
2. Servialize the value into `Vec<u8>` with borsh.